docs/radar.js

```diff
@@ -154,25 +167,25 @@ function radar_visualization(config) {
   }
 
   // partition entries according to segments
-  var segmented = new Array(4);
-  for (var quadrant = 0; quadrant < 4; quadrant++) {
-    segmented[quadrant] = new Array(4);
+  var segmented = new Array(8);
+  for (var quadrant = 0; quadrant < 8; quadrant++) {
+    segmented[quadrant] = new Array(8);
```

Comment:
8 or 4? shall it be a number of rings?

Reply:
8 is the number of segments now. earlier there was only 4 segments.